layers/poi/layer.sql

```diff
@@ -1,11 +1,11 @@
 
-CREATE OR REPLACE FUNCTION global_id_from_imposm(osm_id bigint, osm_type text)
+CREATE OR REPLACE FUNCTION global_id_from_imposm(osm_id bigint)
 RETURNS TEXT AS $$
     SELECT CONCAT(
         'osm:',
-        CASE WHEN osm_id<0
-            THEN CONCAT('relation:', -osm_id)
-            ELSE CONCAT(osm_type, ':', osm_id)
+        CASE WHEN osm_id < -1e17 THEN CONCAT('relation:', -osm_id-1e17)
```

Comment:
can you put some links explaining this construct ?

Reply:
Unfortunately I could not find any mention of `use_single_id_space` in the docs.  
Some hints in the commit that implements it: https://github.com/omniscale/imposm3/commit/3f3c12ece126923b361f14d537eeb2908c3e2dcb